AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -65,6 +67,17 @@
  */
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
+/**
+ * @abstract Return the calculated layout.
+ *
+ * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
```

Comment:
@markmark1 Check out ASImageNodeRoundBorderModificationBlock :)

On 6 Jul 2015 at 10:03:53 pm, markmark1 (notifications@github.com) wrote:

In AsyncDisplayKit/ASDisplayNode+Subclasses.h:

> @@ -65,6 +67,17 @@
>   */
>  @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
>   
> +/**
> + * @abstract Return the calculated layout.
> + *
> + * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
What is the right way to do rounded corners of images or buttons if border radius is slow in objective c
…
—
Reply to this email directly or view it on GitHub.